CONTRIBUTING.md

```diff
@@ -129,35 +149,37 @@ The build will copy artifacts as follows:
 ui -> core/src/main/resources/io/questdb/site/public.zip
 ```
 
-# Testing
+# Before you submit
+
+## Testing
 
 We have a lot of unit tests, most of which are of "integration" type, e.g. test
 starts a server, interacts with it and asserts the outcome. We expect all
-contributors to submit PRs with tests. Please reach out to us via slack if you
-uncertain on how to test, or you think existing test is inadequate and should be
+contributors to submit PRs with tests. Please reach out to us via slack if you're
+uncertain on how to test, or you think an existing test is inadequate and should be
```

Comment:
nit: "could be improved" sounds nicer than "is inadequate and should be removed".
```suggestion
uncertain on how to test, or you think an existing test could be improved.
```